net-analyzer/sslsplit/sslsplit-0.5.4.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic
+
+DESCRIPTION="Transparent SSL/TLS interception"
+HOMEPAGE="https://www.roe.ch/SSLsplit"
+
+LICENSE="BSD-2"
+SLOT="0"
+IUSE="elibc_musl test"
+
+if [[ ${PV} == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/droe/${PN}"
+	EGIT_BRANCH="develop"
+else
+	SRC_URI="https://github.com/droe/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+RDEPEND="
+	elibc_musl? ( sys-libs/fts-standalone )
```

Comment:
We usually put conditional deps after unconditional deps.

Reply:
ok, i'll change it